### PR TITLE
Add cflinuxfs3 to SCF

### PR DESCRIPTION
**Pull request by lightlace**
Change the `role-manifest.yml` to add `cflinuxfs3`.

### CODE REVIEW

**Comment by thardeck** [comment]
Have you tested an app against cflinuxfs3? It should be already supported by all out buildpacks.

File: container-host-files/etc/scf/config/opinions-opensuse42.yml [hunk 1]
```diff
@@ -4,20 +4,24 @@ the-stack-name: &the-stack-name opensuse42
...
     description: "Cloud Foundry Linux-based filesystem"
+  - name: cflinuxfs3
+    description: "Cloud Foundry Linux-based filesystem"
   - name: opensuse42
     description: "openSUSE-based filesystem"
 
 the-rootfses: &the-rootfses
   - "opensuse42:/var/vcap/packages/opensuse42/rootfs.tar"
   - "cflinuxfs2:/var/vcap/packages/cflinuxfs2/rootfs.tar"
+  - "cflinuxfs3:/var/vcap/packages/cflinuxfs2/rootfs.tar"
```
Comment (jimmykarily) [inline container-host-files/etc/scf/config/opinions-opensuse42.yml]: cflinuxfs2 -> cflinuxfs3 ?

File: container-host-files/etc/scf/config/opinions-sle12.yml [hunk 2]
```diff
@@ -4,9 +4,12 @@ the-stack-name: &the-stack-name sle12
 the-bundles: &the-bundles
   'buildpack/sle12':      "buildpack_app_lifecycle/buildpack_app_lifecycle.tgz"
   'buildpack/cflinuxfs2': "buildpack_app_lifecycle/buildpack_app_lifecycle.tgz"
+  'buildpack/cflinuxfs3': "buildpack_app_lifecycle/buildpack_app_lifecycle.tgz"
   docker:                 "docker_app_lifecycle/docker_app_lifecycle.tgz"
 
 the-stacks: &the-stacks
+  - name: "cflinuxfs2"
```
Comment (jimmykarily) [inline container-host-files/etc/scf/config/opinions-sle12.yml]: cflinuxfs2 -> cflinuxfs3 ?

File: container-host-files/etc/scf/config/opinions-sle12.yml [hunk 3]
```diff
@@ -15,9 +18,10 @@ the-stacks: &the-stacks
 the-rootfses: &the-rootfses
   - "sle12:/var/vcap/packages/cf-sle12/rootfs.tar"
   - "cflinuxfs2:/var/vcap/packages/cflinuxfs2/rootfs.tar"
+  - "cflinuxfs2:/var/vcap/packages/cflinuxfs2/rootfs.tar"
```
Comment (jimmykarily) [inline container-host-files/etc/scf/config/opinions-sle12.yml]: same

File: container-host-files/etc/scf/config/opinions-sle12.yml [hunk 4]
```diff
@@ -15,9 +18,10 @@ the-stacks: &the-stacks
...
 
 the-images: &the-images
   - "/var/vcap/packages/cf-sle12/rootfs.tar"
   - "/var/vcap/packages/cflinuxfs2/rootfs.tar"
-
+  - "/var/vcap/packages/cflinuxfs2/rootfs.tar"
```
Comment (jimmykarily) [inline container-host-files/etc/scf/config/opinions-sle12.yml]: same

**Comment by lightlace** [comment]
superseded by #2019

**Pull request closed without merging**